src/core/PathUtil.cpp

```diff
@@ -14,8 +14,11 @@ namespace PathUtil
 		Base::UserLADSPA, Base::DefaultLADSPA, Base::UserSoundfont, Base::DefaultSoundfont, Base::UserGIG, Base::DefaultGIG,
 		Base::LocalDir };
 
-	QString baseLocation(const Base base)
+	QString baseLocation(const Base base, bool* ok /* = nullptr*/)
 	{
+		// ok is true unless something goes wrong
```

Comment:
I believe 0 is used for success in most error codes, so it would be better to have a `bool* error`. That way you can have several different error codes (which all evaluate to true) and one success code (which evaluates to false). If you flip it, you can only have one error, or you sacrifice the ability to use `if(error)`.

Of course this isn't an issue in practice right now, since we only have one error code, but I think it'd be more consistent to follow that convention.

Reply:
I'm flipping it now, should I replace it with a pointer to `int` or can we keep it as a `bool` for now while we don't have different error codes?

Reply:
Either is fine with me, so maybe let's stick with the bool for now. Either way we don't end up in a situation where a bunch of if statements have to be inverted because we want to add more status codes.

Reply:
Cool, it's done then!